src/components/Header/Header.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import s from "./Header.module.css";
+
+
```

Comment:
Consider removing redundant empty lines.
The rule is to have only one empty line that separates blocks of grouped code.

Reply:
fixed